bugsnag-plugin-android-ndk/src/main/assets/include/event.h

```diff
@@ -0,0 +1,291 @@
+#ifndef BUGSNAG_ANDROID_NDK_REPORT_API_H
+#define BUGSNAG_ANDROID_NDK_REPORT_API_H
+
+#include <stdbool.h>
+#include <sys/types.h>
+#ifndef BUGSNAG_METADATA_MAX
+/**
+ * Maximum number of values stored in metadata. Configures a default if not
+ * defined.
+ */
+#define BUGSNAG_METADATA_MAX 128
+#endif
+#ifndef BUGSNAG_FRAMES_MAX
+/**
+ *  Number of frames in a stacktrace. Configures a default if not defined.
+ */
+#define BUGSNAG_FRAMES_MAX 192
+#endif
+#ifndef BUGSNAG_CRUMBS_MAX
+/**
+ *  Max number of breadcrumbs in a report. Configures a default if not defined.
+ */
+#define BUGSNAG_CRUMBS_MAX 30
+#endif
+#ifndef BUGSNAG_DEFAULT_EX_TYPE
+/**
+ * Type assigned to exceptions. Configures a default if not defined.
+ */
+#define BUGSNAG_DEFAULT_EX_TYPE "c"
+#endif
+/**
+ * Version of the bugsnag_report struct. Serialized to report header.
+ */
+#define BUGSNAG_EVENT_VERSION 3
+
+#define BUGSNAG_USER_INFO_LEN 64
+
```

Comment:
Not including a guard for `__cplusplus` is consistent with what was there before. Was that a pre-existing bug? https://github.com/bugsnag/bugsnag-android/pull/637/files#diff-8f292ff3995248bd79217aba9d6c6f2e

I have moved the comment header to `jni/event.h`.

Reply:
The header guards are added to prevent C++ function name mangling breaking interop between C and the JNI. Since there's no functions in this file, no header guards are needed.

Reply:
Resolving as my understanding is that no further action is required here.